src/main/java/Backend.java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Backend {
+    public static ArrayList<Task> initialisation() {
+        ArrayList<Task> res = new ArrayList<Task>();
```

Comment:
Java coding standard: Collection of objects should be denoted by a plural name.

Also consider using a more descriptive name such as `resultingTasks`.